config/environments/development.rb

```diff
@@ -13,6 +13,7 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.